darts/models/exponential_smoothing.py

```diff
@@ -53,6 +53,12 @@ def __init__(self,
             `statsmodels.tsa.holtwinters.ExponentialSmoothing.fit()`.
             See `the documentation
             <https://www.statsmodels.org/stable/generated/statsmodels.tsa.holtwinters.ExponentialSmoothing.fit.html>`_.
+
+        Attributes
+        ----------
+        params
+            The parameters for the Exponential Smoothing model.
```

Comment:
> Sorry, I couldn't know why the docs test fails...
> Could someone please help me?

It seems like these lines are causing buildDocs to fail. Document directly in `params()` instead, e.g.:
```
    @property
    def params(self) -> Dict[Any, Any]:
        """
        The parameters for the Exponential Smoothing model.
        """
        return self._params
```

Though you might want to do that "one level higher" by defining and document a `params()` method in `ForecastingModel`, as @pennfranc suggested.